csrfguard/src/main/java/org/owasp/csrfguard/CsrfGuard.java

```diff
@@ -65,6 +65,11 @@
 
     private Properties properties = null;
 
+    /**
+     * Flag indicating that the JavaScript configuration must be initialised when no configuration is retrieved.
+     */
+    private boolean initialiseJavaScriptConfigurationForNewConfigurationInstances;
```

Comment:
In my understanding, this field name should rather be named something like `isJavaScriptConfigurationNeeded`. That should be clear enough so it wouldn't require a comment either .